clients/redigo.go

```diff
@@ -115,7 +115,23 @@ func (r *Redigo) JSONType(key, path string) (res interface{}, err error) {
 	if err != nil {
 		return nil, err
 	}
-	return r.Conn.Do(name, args...)
+
+	res, command_err := r.Conn.Do(name, args...)
+
+	if err != nil {
+		return nil, command_err
+	}
```

Comment:
Why not use `err` variable itself here ?

Reply:
it said that no variable on the left were created, something like this in my IDE I don't know. I can try again later today

Reply:
![image](https://user-images.githubusercontent.com/13908455/149622714-ad3e2c1d-3539-4f8e-a0f9-c8ba1ca280ca.png)

I think maybe its due to the err declaration above 🤔

Reply:
```go
func (r *Redigo) JSONType(key, path string) (res interface{}, err error) {

	name, args, err := rjs.CommandBuilder(rjs.ReJSONCommandTYPE, key, path)
	if err != nil {
		return nil, err
	}

	res, err = r.Conn.Do(name, args...)

	if err != nil {
		return nil, err
	}
	switch v := res.(type) {
	case string:
		return v, nil
	case []byte:
		return string(v), nil
	case nil:
		return
	default:
		err := fmt.Errorf("type returned not expected %T", v)
		return nil, err
	}
}
```

Changing to just = instead of := it works, sorry, getting used to Go notation hehe

Reply:
Newbie question ❓  @Shivam010  this way the res is already created by the returning definition for the function? 🤔  so no need to var assignment?

Reply:
`:=` declares a new variable and hence, I think you were getting the error. `=` this will just assign the value to already created variables